### DISCUSSION TITLE
FTDI Cable Termination Question

### DISCUSSION THREAD

**Discussion by niharmehta** [aqualinkd/AqualinkD#198] [Q&A]
I am using the FTDI pigtail cable.  On the Wiki it seems to indicate that to use the termination resistor requires just connecting the two specific wires together (brown,green).  However, looking at the  [FTDI Documentation](https://ftdichip.com/wp-content/uploads/2020/07/DS_USB_RS485_CABLES.pdf)  the diagram seems to indicate that they are just wires connected to two sides of the resistor.  So connecting them together may not do anything, but would require to connect one wire to each of the Data wires.  Has anyone confirmed (or tested with a DMM)  proper use of the termination resistors?

**Comment by ballle98** [comment]
see https://www.analog.com/en/technical-articles/rs485-cable-specification-guide--maxim-integrated.html

![image](https://user-images.githubusercontent.com/2252524/221976849-0d8ffc7a-c185-4e87-bfc8-c68f9962536a.png)

In my case I was adding an FTDI device to a control panel attached to a SWG and wireless PDA.  Since it was working before I added the FTDI there must be termination on the 2 ends already so I don't need to add a resistor.  I have probably 30 ft of CAT 5e between the FTDI and the control panel and it seems to work fine.  If there is nothing else attached to the control pannel connector you need to wire in the terminating resistor using the green and brown wires.  If you have signal issues because you have even more cable between the FTDI and the control panel you may want to put the termination at your FTDI and remove the termination from some other device.

![image](https://user-images.githubusercontent.com/2252524/221975794-9ebbeda9-6210-4fd6-a824-568f02249cd4.png)
![image](https://user-images.githubusercontent.com/2252524/221975986-17822962-9a49-4a15-9c35-4978f2ea7a57.png)

**Comment by ballle98** [comment]
![image](https://user-images.githubusercontent.com/2252524/221980104-88adf0d6-d0bc-4785-be61-e6b19703463b.png)

If you need the termination I believe you would wire it in parallel with data+ (Yellow) and data - (orange).  In other words connect yellow and brown to data - on control pannel.  Connect orange and green to the data + on control panel and ground to ground.  You don't want to connect power because you are not supplying power.

**Comment by sfeakes** [comment]
@niharmehta can you link to where the documentation indicates connecting the two wire together and I’ll change it, as that’s not accurate.  You connect them together with a resistor.   You should also never need to do this at the control panel, only at the device end. And even then it’s rarely needed but it all depends on wiring and devices.

**Reply by niharmehta** [reply]
@sfeakes ,  this part in the wiki TLDR may use some clarification.  Although it does not explicitly state to connect those wires together, specifying to connect the wires to the Data +/-  will may be helpful


"So in short, it's best to get an USB2RS485 that will give you the option to ground and the choice to terminate or not. Some adapters do this by the way of allowing you to solder a resistor to the board, and others (as in below link) you can simply connect 2 wires together."

**Comment by sfeakes** [comment]
@niharmehta   Thanks, now I understand.  We are talking about 2 slightly different things.
1 - You need a resistor to terminate the RS485 bus
2a - Some devices have the resistor pre-soldered and can’t easily be removed.
2b - Some devices do not have a resistor
2c - Some devices have the resistor but it’s not connected to the circuit, and to connect it to the circuit you simply join 2 wires.

So in the case of that specific FTDI adapter listed, its (2c), so you do simply connect 2 wires, BUT you are not connecting the Data+ and Data- together, you are connecting them to a resistor that’s already in the circuit.
Page 7 and 8 of the FTDI documentation.
https://ftdichip.com/wp-content/uploads/2020/07/DS_USB_RS485_CABLES.pdf

Hope that helps.
